language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/references/ReferencesTreeVisitor.java

```diff
@@ -101,11 +103,15 @@ public ReferencesTreeVisitor(LSServiceOperationContext context) {
 
     @Override
     public void visit(BLangPackage pkgNode) {
-        if (pkgNode.topLevelNodes.isEmpty()) {
+        List<TopLevelNode> topLevelNodes = new ArrayList<>(pkgNode.topLevelNodes);
+        if (!pkgNode.testablePkgs.isEmpty()) {
+            topLevelNodes.addAll(pkgNode.testablePkgs.get(0).topLevelNodes);
```

Comment:
Even when getting the testable package we can reuse this method: https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangPackage.java#L217?